.github/workflows/discord-announcements.yml

```
@@ -0,0 +1,62 @@
+name: Discord Announcements
+
+on:
+  release:
+    types: [published]
+
+jobs:
+  announce-release:
+    runs-on: ubuntu-latest
+    if: ${{ secrets.DISCORD_ANNOUNCEMENTS_WEBHOOK_URL != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)